crates/sim2h/src/mono_ref.rs

```diff
@@ -0,0 +1,110 @@
+use crate::*;
+
```

Comment:
Can you please add a bit of documentation to describe the intent of the MonoRef and why it's needed.  I think I get it but it would be useful to have a couple lines at the top.